tasks/main.yml

```diff
@@ -103,34 +97,6 @@
       loop_control:
         loop_var: user
         label: "{{ user.name }} {{ user.state }}"
-      when:
-        # Ansible user module and usermod command fail to modify any logged in user when uid is set
-        # Workaround is to run usermod manually for logged in users and not set UID even if it is defined
-        - user.uid is not defined or (user.uid is defined and user.name not in admin_logged_in_users)
-
-    - name: command usermod - primary group and extra groups, workaround for logged in users
```

Comment:
I was not able to replicate the issue for the reason of this work around.

Reply:
This test on CentOS7?

Reply:
Tested with one CentOS7 host on our side and couldn't replicate either. Still want to do a few more tests though.

Reply:
I was now able to replicate the reason for the original workaround. So I was probably testing the wrong thing yesterday. 

Let's say my original UUID was 10000123 and I wanted to change it to 10000321. I set the corresponding `uid: 10000321` in the group_vars dictionary that defines my user.

Lo and behold:

```
TASK [ansible-role-users : Add or remove users from the lists in admin_list_of_user_lists] **********************************************************************************
failed: [HOST] (item=junousia present) => {"ansible_loop_var": "user", "changed": false, "msg": "usermod: user junousia is currently used by process 19297\n", "name": "junousia", "rc": 8, "user": {"groups": "admin", "name": "junousia", "pubkey": "ssh-rsa KEY==", "shell": "/bin/bash", "state": "present", "uid": 10000321}}
```